<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CortexIdentificationDialog</class>
 <widget class="QDialog" name="CortexIdentificationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cortex Identification</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cortex Identification</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="nextTool">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Stage: Scrub Mask</string>
   </property>
  </widget>
  <widget class="QLabel" name="stepNameText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Next step: find the inner cortical boundary</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="restoreFromPreviousSessionButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore From Previous...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tissueThresholdEditBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tissue threshold</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="statusBoxTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="includeAllSubcorticalLabelsCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>19</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="subcortexLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>113</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Include all areas labeled as subcortex as part of initial mask</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
